libraries/engage/product/components/ProductProperties/Content.jsx

```diff
@@ -12,12 +12,12 @@ import { groupsContainer } from './style';
  * @returns {JSX}
  */
 const Content = ({ properties }) => {
+  const groups = useMemo(() => getGroupsFromProperties(properties), [properties]);
```

Comment:
Is there a reason why the groups are evaluated before "!properties return null"

Reply:
yes... react hooks always need to be called in the same order. calling them conditionally could potentially break the hook system. you'll get an eslint warning when moving this line below the properties check.